spec/support/controller_exceptions.rb

```diff
@@ -0,0 +1,23 @@
+module ControllerExceptions; end
+
+# Test group helpers for allowing exceptions from controller tests to be caught by specs.
+module ControllerExceptions::TestGroupHelpers
+  BYPASS_RESCUE_CONSTANT_NAME = :__controller_exceptions_test_group_helpers_bypass_rescue
+
+  def self.extended(module_)
+    module_.module_eval do
+      let(BYPASS_RESCUE_CONSTANT_NAME) { true }
+      before(:each) do
+        bypass_rescue if send(BYPASS_RESCUE_CONSTANT_NAME)
```

Comment:
You get to test what exceptions are thrown from within a controller. The controller is supposed to deal with these exceptions (in the rescue block), but by swallowing them you'd have to check for the effects of handling the exception, and not the exception itself.